packages/bot-web-ui/src/components/bot-footer-extensions/bot-footer-extensions.tsx

```diff
@@ -1,19 +1,11 @@
 import React from 'react';
-import RootStore from 'Stores/index';
-import { connect } from 'Stores/connect';
+import { useStore, observer } from '@deriv/stores';
 import SecurityAndPrivacy from './security-and-privacy';
 
-type TPopulateFooterExtensionsProps = {
-    position: string;
-    Component: React.FC;
-    has_right_separator: boolean;
-};
-
-type TBotFooterExtentionsProps = {
-    populateFooterExtensions: (footer_extensions: TPopulateFooterExtensionsProps[]) => void;
-};
-
-const BotFooterExtensions = ({ populateFooterExtensions }: TBotFooterExtentionsProps) => {
+const BotFooterExtensions = observer(() => {
+    const {
+        ui: { populateFooterExtensions },
+    } = useStore();
```

Comment:
```suggestion
const BotFooterExtensions = observer(() => {
    const { ui } = useStore();
    const { populateFooterExtensions } = ui;
```

We've been destructuring like this on all the other packages for better readability, we should make it consistent accross the app

Reply:
done ✅